tests/integration/cloud/end_to_end/conftest.py

```diff
@@ -67,9 +86,6 @@ def expectation_suite(
     )
     assert len(expectation_suite.expectations) == 0
     yield expectation_suite
-    expectation_suite = context.add_or_update_expectation_suite(
-        expectation_suite=expectation_suite
-    )
```

Comment:
I had to move this into the test modules, because the suite was getting used downstream in tests before it was ever saved.